RELEASE-NOTES.md

```diff
@@ -2,6 +2,12 @@
 
 ## 1.5.0 (dev)
 
+* Added `TermSerializer`
+* Added `TermDeserializer`
+* Added `TermListSerializer`
+* Added `TermListDeserializer`
+* Added `AliasGroupSerializer`
+* Added `AliasGroupDeserializer`
```

Comment:
These classes are not meant to be exposed in public. Only the factories are publicly visible in this component. So only changes on those classes are supposed to appear in the release notes.